changelog.d/239.change.rst

```diff
@@ -0,0 +1,3 @@
+Added ``type`` argument to ``attr.attr()`` and corresponding ``type`` attribute to ``attr.Attribute``.
```

Comment:
`attr.ib()` :)  it can be admittedly confusing when coming from the actual source.  I’ve been thinking to refactor the implicit names for a while…